boards/nano33ble/src/main.rs

```diff
@@ -292,8 +296,8 @@ pub unsafe fn main() {
     let cdc = components::cdc::CdcAcmComponent::new(
         &nrf52840_peripherals.usbd,
         capsules::usb::cdc::MAX_CTRL_PACKET_SIZE_NRF52840,
-        0x2341,
-        0x005a,
+        0x16c0,
+        0x05e1,
```

Comment:
Why change this?

Reply:
Oy... that's a mistake, I had just been playing around and forgot it there. Removing the change, thanks